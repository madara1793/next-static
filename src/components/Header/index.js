// @flow
/* eslint-disable react/no-danger */

import React from 'react';
import config from '../../../config';

export default () =>
  <header className="header">
    <h1 className="header--siteName">{config.siteName}</h1>
    <div className="header--slogan">{config.slogan}</div>
  </header>;
