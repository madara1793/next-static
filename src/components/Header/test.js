import React from 'react';
import { renderToStaticMarkup as render } from 'react-dom/server';
import dom from 'cheerio';
import Header from './index';
import renderer from 'react-test-renderer';

describe('Snapshot::<Header /> component', () => {
  it('should render the contents of the component.', () => {
    const el = <Header />;
    const tree = renderer.create(el).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
