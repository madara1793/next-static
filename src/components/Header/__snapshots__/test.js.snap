// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot::<Header /> component should render the contents of the component. 1`] = `
<header
  className="header Header__Header-te6fwh-0 ldnsLH"
>
  <h1
    className="header--siteName"
  >
    Next Static Blog
  </h1>
  <div
    className="header--slogan"
  >
    ✨ Start static then go server rendered when you need to ✨
  </div>
</header>
`;
